ara/views/playbook.py

```diff
@@ -47,15 +49,15 @@ def playbook_results(playbook):
                     .filter(models.Playbook.id == playbook.id)
                     .order_by(models.TaskResult.time_start))
 
-    if request.args.get('host'):
-        hosts = [str(host) for host in request.args.get('host').split(',')]
+    if host is not None:
+        hosts = [str(h) for h in host.split(',')]
```

Comment:
> I don't think we should be crippling the web ui just to make static generation more convenient.

Agreed.
This is how I got things to work for the time being as I couldn't figure out how to make it work otherwise.
Other options included using [URL generators](http://pythonhosted.org/Frozen-Flask/#url-generators) which I could not get to work properly or outright mostly duplicating the routes in another "app".
Like, instead of @app.route, we could use @static.route, etc:

    @playbook.route('/<playbook>/results')
    @static.route('/<playbook>/results/')
    @static.route('/<playbook>/results/<host>/')
    @static.route('/<playbook>/results/<host>/<status>')

I really like flask frozen though, it has a fair amount of use around the web and I realistically don't see us implementing something similar that's not a crippled (static) web UI without a good amount of work.

I'll give this some more though.